src/Components/Projects.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable max-len */
+/* eslint-disable linebreak-style */
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { InnerWrapper, OuterWrapper, Title, Tags, TagsContainer, Link } from 'StyledComponents/GlobalComponents'
+import { ClonedRedTitle } from './OtherProjects'
+
+const MyProjects = styled.div`
+background: #F4F4F4;
+display: flex;
+flex-direction: column;
+align-content: center;
+align-items: center;
+align-self: center;
+justify-content: center;
+`
+
+const TextContainerProjects = styled.h1`
+font-family: 'Montserrat', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 30px;
+    line-height: 50px;
+    display:grid;
+    grid-template-columns: 1fr;
+   padding: 15px;
+    
+    @media (min-width: 1025px) {
+    grid-template-columns: 1fr 1fr;
+    column-gap: 3vw;
+    }
+`
```

Comment:
Have you tried adding the ImageContainer position to relative and the ImgOverlay position to absolute with a hover of background-color: transparent in order to make the overlay to work? 
Also I had to make the overlay just be on top of the thumbnail as a self-closing tag, and not wrapping around the image for mine to work.. 
Worth a try if you haven't already.